packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -146,38 +158,45 @@ const setupDriver = async () => {
 		}
 	}
 
-	if ( ! isLocalEnvironment() ) {
-		desiredCaps.name = `Gutenberg Editor Tests[${ rnPlatform }]-${ branch }`;
-		desiredCaps.tags = [ 'Gutenberg', branch ];
-	}
-
-	await driver.init( desiredCaps );
+	const sauceOptionsConfig = ! isLocalEnvironment()
+		? {
+				'sauce:options': {
+					...sauceOptions,
+					name: `Gutenberg Editor Tests[${ rnPlatform }]-${ branch }`,
+					tags: [ 'Gutenberg', branch ],
+					idleTimeout: SAUCE_LABS_TIMEOUT,
+				},
+		  }
+		: {};
+	const serverConfig = isLocalEnvironment()
+		? serverConfigs.local
+		: serverConfigs.sauce;
 
-	const status = await driver.status();
-	// Display the driver status
-	// eslint-disable-next-line no-console
-	console.log( status );
+	const driver = await remote( {
+		...serverConfig,
+		logLevel: 'error',
+		capabilities: {
+			platformName: PLATFORM_NAME,
+			...prefixKeysWithAppium( desiredCaps ),
+			...sauceOptionsConfig,
+		},
+	} );
 
 	await driver.setOrientation( 'PORTRAIT' );
 	return driver;
 };
 
 const stopDriver = async ( driver ) => {
 	if ( ! isLocalEnvironment() ) {
-		const jobID = driver.sessionID;
-
-		const hash = crypto
```

Comment:
This was causing issues due to the server configuration changes for SauceLabs. The jobs links are public so I don't think we need that auth param.

Reply:
🤔 I have less familiarity with this infrastructure. Would it make sense to validate this with Apps Infrastructure?

Reply:
After looking at this again I investigated further when it was implemented. This was introduced in this PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/849. It gave me insights into the [SauceLabs Sharable links](https://docs.saucelabs.com/test-results/sharing-test-results/index.html#building-sharable-links).

I checked their latest documentation [about this approach](https://docs.saucelabs.com/test-results/sharing-test-results/#example---nodejs) and I ended up updating the code to keep the existing functionality but with some recent updates from the docs. What do you think about keeping the code as it was (with the recent changes)?

Reply:
Here's the commit with the code update https://github.com/WordPress/gutenberg/pull/55166/commits/9db690e1ba017c0d49372384ef65bab90fb3024a

Reply:
The new approach looks great and feels more appropriate. Well done!